<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <widget class="QLabel" name="lbAddNewBook">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Add New Book</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>821</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QLineEdit" name="edIsbn">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>10</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edTitle">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>40</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edEdition">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>70</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edCopyright">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>100</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edAuthors">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>130</y>
      <width>541</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbIsbn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ISBN</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbTitle">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbEdition">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Edition</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbCopyright">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Copyright</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbAuthors">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Authors</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbPublisher">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Publisher</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbPrice">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Price</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edPublisher">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dspinPrice">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.250000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>1241</width>
     <height>521</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::InternalMove</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::TargetMoveAction</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>220</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
